### PR TITLE
Adding CentOS 8 -> AlmaLinux migration guide 

### PR DESCRIPTION
**Pull request by XargsUK**

### CODE REVIEW

**Comment by XargsUK** [comment]
It failed on spellcheck, but this was due to the use of the terms EOL and AlmaLinux.

File: source/operatingsystems/linux/misc/migrating-centos8-almalinux.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Migrating from CentOS 8 to AlmaLinux
+
+On December 31, 2021, CentOS 8 will reach end-of-life, meaning that Red Hat will no longer make security and maintenance updates to the OS. Since the announcement of this EOL date, several open-source RHEL (Red Hat Enterprise Linux) forks have been created, including AlmaLinux. This guide details the steps for migrating from CentOS 8 to AlmaLinux.
+
+## Prerequisites
+
+* Server running CentOS 8.3 or above. You can verify this by running `cat /etc/redhat-release`
```
Comment (Xiol) [inline source/operatingsystems/linux/misc/migrating-centos8-almalinux.md]: Is it worth adding instructions on how to upgrade to CentOS 8.3? Alternatively could just link them to the [updates page](https://docs.ukfast.co.uk/operatingsystems/linux/basics/packageupdates.html#updating-packages)?

File: source/operatingsystems/linux/misc/migrating-centos8-almalinux.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+## Finishing the Migration
+
+Once the script has finished running, it will display "Migration to AlmaLinux is completed". The last stage is to reboot the server, which will load the AlmaLinux Kernel.
+
+```bash
+sudo reboot
+```
+
+Once your server has rebooted, reconnect via SSH and confirm the version of AlmaLinux:
+
```
Comment (Xiol) [inline source/operatingsystems/linux/misc/migrating-centos8-almalinux.md:47]: Could be worth adding a new "More Information" section and linking them directly to the [migration repo](https://github.com/AlmaLinux/almalinux-deploy) and the [cPanel docs for it](https://support.cpanel.net/hc/en-us/articles/4404770842263-How-to-convert-from-CentOS-8-to-AlmaLinux-8).

A link to the main AlmaLinux website would be useful too.

**Review by Xiol — COMMENTED** [review]
LGTM, just a few suggestions.

**Comment by Xiol** [comment]
RE the spellcheck failures, the failing words can be added to `/.wordlist.txt` to fix that.

**Comment by XargsUK** [comment]
Thanks for additional notes @Xiol, added to the wordlist, additional information added at the bottom linking to vendors and linked to linux update documentation.

**Review by Xiol — APPROVED** [review]

**Comment by Xiol** [comment]
Approved. Need someone with write access to approve and merge, though!

**Review by adamskie — APPROVED** [review]

**Pull request merged**